MailCore/Utils/Array+Extension.swift

```diff
@@ -50,3 +50,9 @@ public extension LazyMapSequence {
         return Array(self)
     }
 }
+
+public extension Set {
```

Comment:
move to core ?

Reply:
More specificaly, can we move the base type extensionn  to core ?
The specific Realm code ca stay here.

public extension List {
public extension LazyFilterSequence {
public extension LazyMapSequence {

Reply:
discussed, will be done in another PR